debian/rules

```diff
@@ -38,7 +38,6 @@ override_dh_auto_install:
 	mkdir $(DESTDIR)/$(SYSCONFDIR)/profile.d
 	mkdir $(DESTDIR)/usr/share/$(PACKAGE)-agent
 	mkdir $(DESTDIR)/usr/share/$(PACKAGE)-agent/plugins
-	install -D agent/target/cloud-agent-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/$(PACKAGE)-agent.jar
 	install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
```

Comment:
```suggestion
	install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/cloud-plugin-hypervisor-kvm-$(VERSION).jar
```
@rhtyd build giving error,
```
install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-4.15.1.0-SNAPSHOT.jar "debian/tmp"/usr/share/cloudstack-agent/lib/
install: target 'debian/tmp/usr/share/cloudstack-agent/lib/' is not a directory: No such file or directory
debian/rules:26: recipe for target 'override_dh_auto_install' failed
```
Adding jar filename should fix this?

Reply:
@shwstppr thnx, the mkdir was missing fixed.